core/src/main/java/org/apache/brooklyn/core/effector/BasicParameterType.java

```diff
@@ -32,6 +32,7 @@
     private Class<T> type;
     private String description;
     private Boolean hasDefaultValue = null;
+    private boolean shouldSanitize = false;
```

Comment:
Sorry, you are right. It is needed only in `EffectorSummary.ParameterSummary`
Check my commit now.